package-lock.json

```diff
@@ -30379,6 +30330,7 @@
       "version": "30.1.1",
       "license": "MIT",
       "dependencies": {
```

Comment:
Not sure why there are so many diffs. Maybe Renovate is using a different npm version?

Last time these changes resulted in a dirty lockfile when I ran npm install against main, so I don't think we want to submit this as-is.

Reply:
Previous time this came up: https://github.com/ni/nimble/pull/2443#issuecomment-2429923791

Reply:
Or at least similar types of issues

Reply:
Rebuilt the lockfile 👍